app/lib/screens/move_screen.dart

```diff
@@ -0,0 +1,94 @@
+import 'dart:async';
+
+
+import 'package:flutter/material.dart';
+import 'package:manvsim/models/patient.dart';
+
+import 'package:manvsim/services/patient_service.dart';
+import 'package:manvsim/widgets/api_future_builder.dart';
+import 'package:manvsim/widgets/logout_button.dart';
+import 'package:manvsim/widgets/timer_widget.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+import '../models/location.dart';
+
+class MoveScreen extends StatefulWidget {
+  final Patient patient;
+  final Location moveTo;
+
+  const MoveScreen(
+      {super.key,
+      required this.patient,
+      required this.moveTo,});
+
+  @override
+  State<MoveScreen> createState() => _MoveScreenState();
+}
+
+class _MoveScreenState extends State<MoveScreen> {
+  late Future<Duration> futureDuration;
+
+  @override
+  void initState() {
+    super.initState();
+    futureDuration = PatientService.movePatient(widget.patient, widget.moveTo);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          backgroundColor: Theme.of(context).colorScheme.inversePrimary,
+          title:  Text(AppLocalizations.of(context)!.moveScreenTitle(
+              widget.patient.name,
+              widget.patient.location.name,
+              widget.moveTo.name)),
+          actions: const <Widget>[LogoutButton()],
+          automaticallyImplyLeading: false,
+        ),
+        body: Center(
+            child: ApiFutureBuilder<Duration>(
+                future:  futureDuration,
+                builder: (context, waitTime) {
+                  return TimerWidget(
+                    duration: waitTime,
+                    onTimerComplete: () =>
+                        showResultDialog(successContent()),
+                  );
+                },
+                onError: () =>
+                    Timer.run(() => showResultDialog(failureContent())))));
+  }
+
+  void showResultDialog(Widget content) {
```

Comment:
We have significant code duplication with action_screen here. Somehow unite them? Or put methods into a util file?  
Although i guess for actions it may be simplifies, because the sucess content isn't supposed to show in a dialog. Same for Move or dialog there?

Reply:
the stucture is similar but there are diferences in code, that will become even more when implementing the multimedia view for the actions. I would suggest to keep the separate file